tests/appsec/waf/test_telemetry.py

```diff
@@ -77,7 +77,7 @@ def test_metric_waf_init(self):
 
     setup_metric_waf_updates = _setup
 
-    @missing_feature(reason="Test not implemented")
+    @irrelevant(reason="Test not implemented")
```

Comment:
`missing_feature` is the good decorator, with the key principle : test is part of the feature.

Though, I wonder why there is such a comment. I mean, I can read some code

Reply:
I'm checking with Santi about it...

Reply:
test removed.